### DISCUSSION TITLE
Can we use dynamodb in sequelize?

### DISCUSSION THREAD

**Discussion by vkamsingh** [sequelize/sequelize#15811] [Help & Questions] [answered]
Hello, is it possible to use Dynamodb in sequelize?

**Comment by WikiRik** [comment, marked as answer]
dynamodb is currently not supported by sequelize, nor are there any plans to support a NoSQL database engine in sequelize v7.

We have planned for sequelize v8 however to make it easier to implement unofficial dialects (i.e. a dialect that would be supported by the community instead of the sequelize team). That may be a viable solution for new dialects if there is not enough demand for it to be an officially supported dialect. For now it is unclear if NoSQL will be easier to implement in sequelize v8 or only new SQL dialects.
